convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -146,13 +147,35 @@ def run(self):
                 latest_available_version = (package_version[1], package_version[2], package_version[3])
 
         logger.debug("Found %s to be latest available version" % (latest_available_version[1]))
+
+        # Get source files that we’re running with import convert2rhel ; convert2rhel.__file__
+        running_convert2rhel_init_file = convert2rhel_file
+
+        # Run `rpm -qf <source file>` to get the installed convert2rhel package NEVRA
+        running_convert2rhel_NEVRA, return_code = utils.run_subprocess("rpm", "-qf", running_convert2rhel_init_file)
+
+        # Run `rpm -V <convert2rhel pkg NEVRA>` to make sure the user hasn’t installed a different convert2rhel version on top of a previously installed rpm package through other means than rpm (e.g. pip install from GitHub)
+        correct_rpm_convert2rhel_version, return_code1 = utils.run_subprocess("rpm", "-V", running_convert2rhel_NEVRA)
```

Comment:
`correct_rpm_convert2rhel_version` is going to be a string because that's what `run_subprocess()` returns.  However, the rest of the code expects that the version that is being compared will be a tuple of strings:
```
(epoch, version, release)
 ("0", "1.3.0", "1")
```

We should be able to use a function that Preston wrote to parse the string: `pkghandler.parse_pkg_string()`
You will then have to select the fields that you need from the parsed string (epoch, vresion, and release) to fill in the version that you use for the rest of the code.

Reply:
done!